### PR TITLE
G-API: Refactor render tests

### PR DESCRIPTION
**Pull request by TolyaTalamanov**
build_gapi_standalone:Linux x64=ade-0.1.1f
build_gapi_standalone:Win64=ade-0.1.1f
build_gapi_standalone:Mac=ade-0.1.1f
build_gapi_standalone:Linux x64 Debug=ade-0.1.1f

### CODE REVIEW

**Comment by TolyaTalamanov** [comment]
@smirnov-alexey @dmatveev

**Comment by smirnov-alexey** [comment]
Looks pretty polished to me

File: modules/gapi/src/api/render_ocv.cpp [hunk 1]
```diff
@@ -92,60 +89,60 @@ void drawPrimitivesOCV(cv::Mat &in, const Prims &prims)
...
+                cv::line(in, lp.pt1, lp.pt2, color, lp.thick);
                 break;
             }
 
             case Prim::index_of<Mosaic>():
             {
-                const auto& l_p = cv::util::get<Mosaic>(p);
-                mosaic(in, l_p.mos, l_p.cellSz);
+                const auto& mp = cv::util::get<Mosaic>(p);
+                GAPI_Assert(mp.decim == 0 && "Only decim = 0 supported now");
```
Comment (smirnov-alexey) [inline modules/gapi/src/api/render_ocv.cpp]: Actually, only ref version doesn't support `decim`, the original one does. So, please, put a fixme here and let's fix it later (if needed)

Comment (TolyaTalamanov) [reply modules/gapi/src/api/render_ocv.cpp]: Fixed

**Comment by TolyaTalamanov** [comment]
@dmatveev

File: modules/gapi/include/opencv2/gapi/render/render.hpp [hunk 2]
```diff
@@ -121,8 +110,6 @@ struct Poly
     std::vector<cv::Point> points;  //!< Points to connect
     cv::Scalar             color;   //!< The line color
     int                    thick;   //!< The thickness of line
-    int                    lt;      //!< The Type of the line. See #LineTypes
-    int                    shift;   //!< The number of fractional bits in the point coordinates
```
Comment (dmatveev) [inline modules/gapi/include/opencv2/gapi/render/render.hpp]: Where did all this go?

**Comment by dmatveev** [comment]
Standalone builds are not tested

**Comment by TolyaTalamanov** [comment]
build_gapi_standalone:Linux x64=ade-0.1.1f
build_gapi_standalone:Win64=ade-0.1.1f
build_gapi_standalone:Mac=ade-0.1.1f
build_gapi_standalone:Linux x64 Debug=ade-0.1.1f

File: modules/gapi/include/opencv2/gapi/render/render.hpp (hunk 2 above)
Comment (TolyaTalamanov) [reply modules/gapi/include/opencv2/gapi/render/render.hpp]: Returned

**Comment by TolyaTalamanov** [comment]
@dmatveev

**Comment by TolyaTalamanov** [comment]
@dmatveev Look at it again and go merge it :)

**Pull request merged**